utils/theme.ts

```diff
@@ -95,15 +97,15 @@ const FormLabel = {
 };
 
 const Heading = {
+  baseStyle: {
+    fontWeight: 400,
+  },
   defaultProps: {
-    variant: 'secondary',
+    variant: 'primary',
   },
   variants: {
     primary: {
-      fontFamily: `'Orbitron', sans-serif`,
-    },
-    secondary: {
-      fontFamily: `'Press Start 2P', cursive`,
+      fontFamily: `'Unbounded', sans-serif`,
```

Comment:
"Unbounded" seems seems great as a body font, but feels a little lacking as a heading in giving people a vague-ish impression that they are playing a video game. Wondering if there's a font out there that gives the same feeling as "Press Start 2P," but isn't as aggressive or ineligible

Reply:
The thing is that the font Unbounded wasn't loading * *facepalm*, so what you were seeing wasn't as intended. See the difference below. I really hope you can reconsider and get on board with this font, bcs I think it is pretty perfect for what we're trying to do in this design. 

**What you saw:**
<img width="1440" alt="Screenshot 2023-10-12 at 09 25 00" src="https://github.com/raid-guild/character-sheets-frontend/assets/51742196/c31d0181-a112-460a-95af-859a729ab86a">

**What it should be (and is now, I fixed the google fonts):**
<img width="1440" alt="Screenshot 2023-10-12 at 09 31 40" src="https://github.com/raid-guild/character-sheets-frontend/assets/51742196/2db0e6d7-9547-4e67-8f4a-646aa6e36cb2">